buildstockbatch/schemas/v0.3.yaml

```
@@ -47,8 +47,10 @@ hpc-postprocessing-spec:
   time: int(required=True)
   n_workers: int(required=False)
   node_memory_mb: enum(85248, 180224, 751616, required=False)
+  n_procs: enum(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, required=False)
```

```suggestion
  n_procs: int(min=1, max=10, required=False)
```

see https://github.com/23andMe/Yamale#integer---intminint-maxint

---

Accepted.